domains/analytical/backend/src/test/AnalyticalSchema.test.ts

```diff
@@ -123,22 +123,15 @@ describe("AnalyticalSchema", () => {
     const elementId: Id64String = iModelDb.elements.insertElement(elementProps);
     // test forEachProperty and PropertyMetaData.isNavigation
     const element: GeometricElement3d = iModelDb.elements.getElement(elementId);
-    element.forEachProperty((propertyName: string, meta: PropertyMetaData) => {
-      switch (propertyName) {
+    const metadata = await element.getMetaData();
+    (await metadata.getProperties()).forEach((property) => {
+      switch (property.name) {
         case "model":
-          assert.isTrue(meta.isNavigation);
-          break;
         case "category":
-          assert.isTrue(meta.isNavigation);
-          break;
         case "typeDefinition":
-          assert.isTrue(meta.isNavigation);
-          break;
         case "codeValue":
-          assert.isFalse(meta.isNavigation);
-          break;
         case "userLabel":
-          assert.isFalse(meta.isNavigation);
+        assert.isTrue(property.isNavigation());
```

Comment:
how can all the properties be navigation properties?

Reply:
oooh my mistake, I thought to merge those many case blocks and didn't realize some expect false :-) I wonder why this test does not fail. Does it not run during rush cover? I'll check.

Reply:
fixed